CMakeLists.txt

```diff
@@ -186,6 +186,13 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
         # in some builds we appear to be missing libz for some strange reason...
 	# Add ssh2 at the end for openSUSE builds (for recent cmake?)
         set(SUBSURFACE_LINK_LIBRARIES ${SUBSURFACE_LINK_LIBRARIES} -lssh2 -lz -lpthread)
+
+        # Test for ARM processor (Raspberry Pi) and add libGLESv2 if found
+        if (CMAKE_SYSTEM_PROCESSOR STREQUAL "armv7l" OR CMAKE_SYSTEM_PROCESSOR STREQUAL "armv6l")
+                message (STATUS "Found ARM processor. Adding libGLESv2")
+                set(SUBSURFACE_LINK_LIBRARIES ${SUBSURFACE_LINK_LIBRARIES} -lGLESv2)
```

Comment:
Is that true for ALL arm v7l/v6l systems? Or should there be an additional test for being on a Raspberry?
I don't know enough about the myriad of ARM systems out there. But let's assume I build on Android system - it seems there we don't need that additional library. (then again, I haven't built Subsurface on Android in a long time, these days all I test is Subsurface-mobile).